package.json

```diff
@@ -45,6 +45,6 @@
     }
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=18.0.0"
```

Comment:
You will need to update the `.github/workflows/main.yml` accordantly.

Reply:
Let me do it 👍